index.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
   
-  <form action="form.html" method="POST">
+  <form target="_self">
```

Comment:
On this exercise we want you to handle this keypress by hand coding it, so no `form` tags.